apps/meteor/client/components/UserStatusMenu.tsx

```diff
@@ -58,7 +78,7 @@ const UserStatusMenu = ({ onChange, optionWidth = undefined, initialStatus = 'of
 
 	return (
 		<>
-			<Button ref={ref} small square ghost onClick={onClick} onBlur={hide} onKeyUp={handleKeyUp} onKeyDown={handleKeyDown} {...props}>
+			<Button ref={ref} small square ghost onClick={onClick} onBlur={hide} onKeyUp={handleKeyUp} onKeyDown={handleKeyDown} margin={margin}>
```

Comment:
👀 what happened to the other props?

Reply:
No other props are passed except for margin. So didn't make sense using props. Just flattened it. If need to extend in the future then we can adjust.